mobile/src/test/java/org/openhab/habdroid/util/UtilTest.java

```diff
@@ -71,15 +88,48 @@ private Document createSitemapDocument() throws ParserConfigurationException, IO
     public void sitemapExists() throws Exception {
         assertTrue(Util.sitemapExists(sitemapList(), "garden"));
         assertFalse(Util.sitemapExists(sitemapList(), "monkies"));
+        assertTrue(Util.sitemapExists(Util.parseSitemapList(createJsonArray(1)), "demo"));
```

Comment:
These lines are very hard to interpret why you sometime expect true, and sometimes false. Could you add amessage to the assert functions and explain it a bit? :)